.cspell.json

```diff
@@ -558,6 +572,7 @@
     "forestgreen",
     "fPIC",
     "fprofile",
+    "fq_codel",
```

Comment:
I'm not sure if it works with `_`. Have you confirmed the behavior?

Seeing the log, I guess it might be `codel`.
```
2022-07-06T22:35:06.4538505Z docs/start-guide/installation/system-setup-host.md:69:71 Unknown word (codel)
```

```suggestion
    "codel",
```

Reply:
Yes, I confirmed it, and it works with "fq_codel". And the error mentioned "fq_codel", like the below image:

![Screen Shot 2022-07-07 at 19 20 28](https://user-images.githubusercontent.com/89968601/177751545-34321253-80b3-46e9-9969-03a7a1712343.png)


Before adding "fq_codel", cspell was printing an error, like the below image:

![Screen Shot 2022-07-07 at 19 16 35](https://user-images.githubusercontent.com/89968601/177751112-cd81f3a3-67e5-4c57-875a-555c1f1821bb.png)

After adding "fq_codel", cspell does not show the error, like the below image:

![Screen Shot 2022-07-07 at 19 16 52](https://user-images.githubusercontent.com/89968601/177751272-475632a8-32ae-4b7f-b978-3226f2d4e903.png)